src/RVRP_data.jl

```diff
@@ -80,12 +80,16 @@ mutable struct Request # can be
     delivery_time_windows::Vector{FlexibleRange}
 end
 
-mutable struct VehicleCategory
-    id::String
-    vehicle_capacities::Dict{String,Float64} # defined only if measured at the vehicle level; for string id key associated with properties capacity measures that need to be checked on the vehicle, as for instance weight, value, volume
-    compartment_capacities::Dict{String,Dict{String,Float64}} # defined only if measured at the compartment level; for string id key associated with properties capacity measures that need to be checked on the vehicle, as for instance weight, value, volume, ... For each such property, the Dictionary specifies the capacity for each compartment id key.
+mutable struct VehicleCapacity
+    vehicle_capacity_measures::Dict{String,Float64} # defined only if measured at the vehicle level; for string id key associated with properties capacity measures that need to be checked on the vehicle, as for instance weight, value, volume
+    compartment_capacity_measures::Dict{String,Dict{String,Float64}} # defined only if measured at the compartment level; for string id key associated with properties capacity measures that need to be checked on the vehicle, as for instance weight, value, volume, ... For each such property, the Dictionary specifies the capacity for each compartment id key.
     vehicle_properties::Dict{String,Float64} # defined only if measured at the vehicle level; for string id key associated with properties that need to be checked on the vehicle (such as the same check applies to all the compartments), as for instance to ability to cary liquids or  refrigerated product.
     compartment_properties::Dict{String,Dict{String,Float64}} #  defined only if measured at the compartment level; for string id key associated with properties that need to be check on the comparments such as  max weight, max length, refrigerated product, .... For each such property, the Dictionary specifies the capacity for each compartment id key.
+end
+
+mutable struct VehicleCategory
```

Comment:
@IssamT  @FD-V 
If we are to split Vehicle Category it may be better do something like the suggestion below. Of course names may be changed.

usage: ` vehicle_category.capacities.of_compartments["weight"]["compartment_1"]`

``` julia
mutable struct VehicleCharacteristic
    of_vehicle::Dict{String,Float64} # put good comments here
    of_compartments::Dict{String,Dict{String,Float64}} # put good comments here
end

mutable struct VehicleCategory
    id::String
    capacities::VehicleCharacteristic # put good comments here
    properties::VehicleCharacteristic # put good comments here
    loading_option::Int # 0 = no restriction (=default), 1 = one request per compartment, 2 = removable compartment separation (note that product conflicts are measured within a compartment)
    energy_interval_lengths::Vector{Float64} # at index i, the length of the i-th energy interval. empty if no recharging.
end
```